shell/src/main/java/org/apache/accumulo/shell/commands/ActiveScanIterator.java

```diff
@@ -71,9 +71,9 @@ private void readNext() {
 
     final String header = String.format(
         " %-21s| %-21s| %-9s| %-9s| %-7s| %-6s|"
-            + " %-8s| %-8s| %-10s| %-20s| %-10s| %-10s | %-20s | %s",
+            + " %-8s| %-8s| %-10s| %-20s| %-10s| %-10s | %-20s | %s | %s",
```

Comment:
Because these are being printed... I'm wondering if it's better to use empty strings instead of nulls wherever null is being passed for the default here. This also makes me wonder if metadata scans internal to Accumulo should have a unique, generated user data.

Reply:
This should be using an empty string and not null now